main.py

```diff
@@ -0,0 +1,11 @@
+from sorting_methods import mergesort, bubblesort
+
+if __name__ == "__main__":
+  list = [4, 3, 5, 7, 43, 2, 5, 6, 87, 4, 3, 1, 3, 4, 5, 6, 7, 45, 3, 2]
```

Comment:
to lista rozruchowa, ale wiedz że  nie powinieneś testować na tak ubogim zbiorze. jakieś floaty, jakieś ujemne, jakieś zera- na przyszłość.